game/scripts/npc/abilities/ember_spirit_flame_guard.txt

```diff
@@ -24,7 +26,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "80 90 100 110 220 350"
+    "AbilityManaCost"                                     "80 90 100 110 220 330"
```

Comment:
I think this is a pointless change. Ember doesn't and shouldn't have mana issues that far late game. I would just leave it as it is tbh